functions/pe_profiles_on_host.pp

```diff
@@ -10,7 +10,7 @@ function puppet_operational_dashboards::pe_profiles_on_host() >> Array[String] {
   $hosts = puppetdb_query("resources[title] {
       type = 'Class' and
       certname = '${trusted['certname']}' and
-      title in ['Puppet_enterprise::Profile::Puppetdb', 'Puppet_enterprise::Profile::Master', 'Puppet_enterprise::Profile::Database'] and
+      title in ['Puppet_enterprise::Profile::Puppetdb', 'Puppet_enterprise::Profile::Master', 'Puppet_enterprise::Profile::Database', 'Puppet_enterprise::Profile::Orchestrator'] and
```

Comment:
for future enhancements I recommend matching on `~` `Puppet_enterprise::Profile` in the future.